src/analysis/modals/AnalysisModal.ts

```diff
@@ -280,6 +282,7 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
       GALAXY: img({ src: galaxyLogo, alt: 'Create new Galaxy app', style: _.merge(styles.image, { width: 139 }) }),
       CROMWELL: img({ src: cromwellImg, alt: 'Create new Cromwell app', style: styles.image }),
       HAIL_BATCH: img({ src: hailLogo, alt: 'Create new Hail Batch app', style: styles.image }),
+      WDS: img({ src: undefined, alt: 'Create new WDS app', style: styles.image }),
```

Comment:
Does this mean that WDS will now appear in the tool sidebar? My understanding is that it only appears in the DATA tab, not the analysis one

Reply:
That's the type of thing I wanted IA review on. I didn't see WDS in the "Start analysis" sidebar after making this change, but I don't know what all these app constants control.

<img width="421" alt="Screenshot 2023-07-17 at 1 04 40 PM" src="https://github.com/DataBiosphere/terra-ui/assets/1156625/47fcbce1-09ce-4cf6-9b97-d988f117e656">

Reply:
You will need to add your `AppTool` to `cloudAppTools` in `tool-utils.ts` for it to show up. `appToolButtons` is where this bubbles up to in `AnalysisModal.ts`

Reply:
Filed [IA-4438](https://broadworkbench.atlassian.net/browse/IA-4439) to add the WDS app type.